models/topics-models.js

```diff
@@ -0,0 +1,18 @@
+const db = require ('../db/connection')
+
+
+
+
+
+exports.fetchAllTopics = () => {
+
+    return db.query('SELECT * FROM topics;').then((result) => {
+        const topics = result.rows;
+
+        if (topics.length === 0) {
+            throw { status: 404, msg: 'No topics found' }
+            }      
+               
```

Comment:
This error handler isn't needed as there are always going to be topics in the database, but nice you are thinking about it! 😄